configure.ac

```diff
@@ -590,6 +590,10 @@ case $host in
      archive_cmds_CXX="\$CC -shared \$libobjs \$deplibs \$compiler_flags -static -o \$output_objdir/\$soname \${wl}--enable-auto-image-base -Xlinker --out-implib -Xlinker \$lib"
      postdeps_CXX=
 
+     dnl We require at least Windows 7 (NT 6.1)
+     AX_CHECK_LINK_FLAG([[-Wl,--major-subsystem-version -Wl,6]], [LDFLAGS="$LDFLAGS -Wl,--major-subsystem-version -Wl,6"],, [[$LDFLAG_WERROR]])
```

Comment:
Would it be possible to combine these into one ? Specifying the one doesn't make sense without the other, after all

Reply:
Sure. I've merged them into a single check.

Reply:
Thanks!